sass/blocks/contact/_contact.scss

```diff
@@ -0,0 +1,61 @@
+.contact {
```

Comment:
![image](https://user-images.githubusercontent.com/1570578/34326335-afe127ce-e8a1-11e7-9018-dd78a109acfe.png)
I don't think you need to create a new block for every form element. I think we can reuse a lot of code. Your inputs styling is the same  What do you think about having just one form block! and all other fields are elements!? I want us not to forget this https://cssguidelin.es/#object-orientation